client/src/util/camera.ts

```diff
@@ -1,18 +1,20 @@
 import { vec2, mat3 } from "gl-matrix";
 import { MouseEvent } from "react";
+import { Point, Viewer } from "openseadragon";
 import clamp from "./clamp";
+import { THROTTLE_MS } from "./constants";
 
 const EPSILON = 0.000001;
 
 const scaleSpeed = 0.5;
 // exporting this for testing
 export const scaleMax = 12.0;
 const scaleMin = 0.5;
-const panBound = 0.8;
```

Comment:
panBound is removed because the pan bound restriction doesn't work well with openseadragon layer and pretty difficult to get both embedding and the image in sync